#! /bin/sh /usr/share/dpatch/dpatch-run
## holux-m241.dpatch by Bernd Zeimetz <bzed@debian.org>
##
## DP: Adding hotplug support for the Holux m241 (and probably others).

@DPATCH@
diff -urNad gpsd~/debian/gpsd.udev gpsd/debian/gpsd.udev
--- gpsd~/debian/gpsd.udev	2008-03-15 14:44:41.000000000 +0100
+++ gpsd/debian/gpsd.udev	2008-07-15 22:16:47.000000000 +0200
@@ -33,6 +33,9 @@
 SYSFS{idVendor}=="067b", SYSFS{idProduct}=="aaa0", SYMLINK="gps%n", RUN+="/lib/udev/gpsd.hotplug.wrapper"
 # Garmin International GPSmap, various models (tested with Garmin GPS 18 USB)
 SYSFS{idVendor}=="091e", SYSFS{idProduct}=="0003", SYMLINK="gps%n", RUN+="/lib/udev/gpsd.hotplug.wrapper"
+# Cygnal Integrated Products, Inc. CP210x Composite Device (Used by Holux m241)
+SYSFS{idVendor}=="10c4", SYSFS{idProduct}=="ea60", SYMLINK="gps%n", RUN+="/lib/udev/gpsd.hotplug.wrapper"
+
 
 ACTION=="remove", RUN+="/lib/udev/gpsd.hotplug.wrapper"
 
